templates/chat/item/item-description.hbs

```diff
@@ -0,0 +1,422 @@
+<section class="chat-item-description">
+    <header class="chat-item-header">
+        <img src="{{item.img}}" alt="">
+        <h4>{{item.name}}</h4>
+    </header>
+    <div class="chat-description">
+        {{#if (or (eq type 'armor') (eq type 'weapon') (eq type 'diagrams') (eq type 'mount') (eq type 'valuable'))}}

```

Comment:
So there a basically several templates in one. Can you solve this via partials instead of a big file with several checks?